portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,9 @@
+package com.google.sps.data;
+
+public final class Comment {
+  private final String message;
+
+  public Comment(String message) {
+    this.message = message;
+  }
+}
```

Comment:
Why do you put this class in a separate file? If it's only in DataServlet.java, it's better to be created there. Besides, if this comment is only a string literal, there is no need to define this class.